src/blocks/stars.css

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+  height: 16px;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background: url(/images/star.svg) no-repeat center;
+  margin-right: 4px;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(/images/star-active.svg);
+}
```

Comment:
remain all selectors (for all amount variants)
what if I will change class modifier through js?
<img width="1440" alt="Screenshot 2022-08-12 at 17 16 29" src="https://user-images.githubusercontent.com/94289038/184372407-f4475370-6db4-43d4-bacd-a80bf8923e5c.png">